homeassistant/components/jellyfin/const.py

```diff
@@ -0,0 +1,36 @@
+"""Constants for the Jellyfin integration."""
+
+DOMAIN = "jellyfin"
```

Comment:
It's good to annotate constants with `typing.Final`.

Reply:
Done